### ISSUE TITLE
`make docker-build` and `make docker` do not work on 9.0.2 branch

### ISSUE THREAD

**Issue by werdnum** [louketo/louketo-proxy#541]
`make docker-build` does not work, because it is hardcoded to Go 1.10, which doesn't seem to have support for pulling dependencies. Updating `GOVERSION` to 1.14.2 in `Makefile` does the trick.

`make docker` does not work because there is no `Dockerfile`.

A minimal working `Dockerfile` might look like this (if you add the `static` target as a dependency for `docker` in `Makefile`)

```
FROM alpine:3.11
ADD bin/keycloak-gatekeeper /usr/bin/keycloak-gatekeeper
ENTRYPOINT /usr/bin/keycloak-gatekeeper
```

**Comment by jangaraj** [comment]
You don't even need `FROM alpine:3.11`. Make static binary and you can use `FROM scratch`.

**Comment by ASzc** [comment]
Should be addressed by #638
